emperor/support_files/js/controller.js

```diff
@@ -202,6 +202,8 @@ EmperorController.prototype.buildUI = function() {
   //FIXME: This only works for 1 scene plot view
   this.colorController = this.addTab(this.sceneViews[0].decViews,
                                      ColorViewController);
+  this.colorController = this.addTab(this.sceneViews[0].decViews,
```

Comment:
`EmperorController` is very much in flux, @mortonjt and I considered that being able to keep a reference to each of the tabs would be useful, specifically once we start thinking about predefined setups i.e. start "start coloring by DOB, and change opacity of treatment to 0.2".

As a side note, I've found that having access to the controllers is useful when debugging.